pronto_mia/lib/core/services/push_notification_service.dart

```diff
@@ -145,32 +146,20 @@ class PushNotificationService {
       subTitle: '$availableFromFormatted - $availableUntilFormatted',
     );
 
-    final dialogResponse = await _dialogService.showCustomDialog(
+    // ignore: prefer_function_declarations_over_variables
+    final AsyncCallback navigationCallback = () async {

```

Comment:
instead of ignoring the linter error i suggest you define a private method in the service called `openPdf` or `navigateToPdf`